docs/sources/datasources/loki/configure-loki-data-source.md

```diff
@@ -116,7 +116,7 @@ Each derived field consists of the following:
 
   - **Regex**: A regular expression to parse a part of the log message and capture it as the value of the new field. Can contain only one capture group.
 
-  - **Label**: A label from the selected log line. This can be any type of label - indexed, parsed or structured metadata. The label's value will be used as the value of the derived field.
+  - **Label**: A label from the selected log line. This can be any type of label - indexed, parsed or structured metadata. When using this type, the input will match as a regular expression against label keys, allowing you to match variations like `traceid` and `trace_id` with a single regex pattern like `trace[_]?id`. The value of the matched label will be used as the value of the derived field.
 
```

Comment:
Should we say anything about being careful with the regex as it could impact rendering performance? I think we have a previous occurrence of this.

Reply:
Good call! Added a note.